app/views/swapsea_mailer/offer_received.html.erb

```diff
@@ -129,14 +129,21 @@
           Status: <%= @request.status.capitalize %>
         </p>
         <p>
+          <% if @offer.roster.present? %>
           <strong>Current offers</strong><br/>
           <% @request.offers.where(:status => 'pending').order(id: :desc).each do |offer| %>
           <div class="offer<%= ' offer-new' if offer.id == @offer.id %>">    
           <%= raw('<strong>NEW</strong><br/>') if offer.id == @offer.id %>
+
+          
             <div class="roster"><%= offer.roster.start.strftime("%A %d %B %Y") %>, <%= offer.roster.start.strftime("%H:%M") %> - <%= offer.roster.finish.strftime("%H:%M") %></div>
+           
             <div class="patrol"><%= offer.roster.patrol_name %></div>
             <div class="member"><%= offer.user.first_name %> <%= offer.user.last_name %></div>
           </div>
+           <% end %>
+           <% else %>
+           No Patrol Offered
```

Comment:
Let's put some styling on "No Patrols Offered" so it looks good.
Also, there is an "end" and "else" on consecutive lines... perhaps had an line comment to make it easier for the next person.